packages/web-frontend/src/actions.js

```diff
@@ -138,6 +138,8 @@ export const SET_PROJECT = 'SET_PROJECT';
 export const FETCH_PROJECTS_ERROR = 'FETCH_PROJECTS_ERROR';
 export const REQUEST_PROJECT_ACCESS = 'REQUEST_PROJECT_ACCESS';
 export const UPDATE_HISTORY_LOCATION = 'UPDATE_HISTORY_LOCATION';
+export const UPDATE_MEASURE_CONFIG_ONCE_HIERARCHY_LOADS =
```

Comment:
This feels really hacky to me...
The reason that I thought creating a new action is necessary:
- We need to wait for the measure hierarchy to be populated before we try to update it
- I only know how to do that in sagas

Reply:
I'll admit my ignorance regarding sagas that must run in sync, @rohan-bes any suggestions for a clean approach here?

Reply:
But if we keep this, I suggest renaming it to `UPDATE_MEASURE_DATE_RANGE_ONCE_HIERARCHY_LOADS` to make our intentions clearer

Reply:
Didn't get a chance to properly look into this one unfortunately, but I can't immediately see a better way of doing it. I'm cool with this going in as it is for now, we always have the option to refactor later.

Reply:
Thanks @rohan-bes !